src/main/scala/lms/thirdparty/random_data.scala

```diff
@@ -0,0 +1,44 @@
+package lms.thirdparty
+
+import lms.core._
+import lms.util._
+import lms.core.stub._
+import lms.core.Backend._
+import lms.core.virtualize
+import lms.core.utils.time
+import lms.macros.SourceContext
+
+object RandomDataTypeLess {
```

Comment:
It seems to be a good idea to also have support for `srand`, something translated to `srand((unsigned) time(0))`.

Reply:
Added